tests/utils.go

```diff
@@ -1435,3 +1437,25 @@ func GetNodeWithHugepages(virtClient kubecli.KubevirtClient, hugepages k8sv1.Res
 	}
 	return nil
 }
+
+// SkipIfVersionBelow will skip tests if it runs on an environment with k8s version below specified
+func SkipIfVersionBelow(virtClient kubecli.KubevirtClient, message string, version string) {
+	response, err := virtClient.RestClient().Get().AbsPath("/version").DoRaw()
+	Expect(err).ToNot(HaveOccurred())
+
+	var info k8sversion.Info
+
+	err = json.Unmarshal(response, &info)
+	Expect(err).ToNot(HaveOccurred())
+
+	gitVersion := strings.Split(info.GitVersion, "+")[0]
+	curVersion := strings.Split(gitVersion, ".")
+	expectedVersion := strings.Split(version, ".")
+	Expect(len(expectedVersion)).To(Equal(len(curVersion)))
```

Comment:
I think that you can simply use `<` on `gitVersion`. Just make sure that you strip te leading `v`.

E.g. `"v1.9" < "v1.20.3"`

Reply:
done